float32[] bl_to_obj_matr
float32[] obj_to_gripper_aa_vector
---
float64[] joint_values
